benchmarks_test.go

```diff
@@ -0,0 +1,75 @@
+package rocket_test
+
+import (
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/dannypsnl/rocket"
+)
+
+var rk *rocket.Rocket
+
+func TestMain(m *testing.M) {
+	rk = rocket.Ignite(":8080")
+	m.Run()
+}
+
+func BenchmarkWithoutUserDefinedContext(b *testing.B) {
+	rk = rk.Mount("/home", rocket.Get("/", func() string {
```

Comment:
I would say just mount in Test Main

Reply:
I like this way more than put it in TestMain, because I can easily know everything I need to know.
If I mount in TestMain, I need to scroll my window to know the context.

Reply:
Yes and I think we should do integration test at others package. I would go to approve this or.